src/api/tasks/TaskCollaborators.js

```diff
@@ -149,6 +151,50 @@ class TaskCollaborators extends TasksBase {
         });
     }
 
+    getUrlForNotificationAnalytics(): string {
```

Comment:
Should we move portions of this logic to a separate API class? it seems to be a new resource not directly related to tasks or collaborators.